### ISSUE TITLE
Update PHP to version 7.2

### ISSUE THREAD

**Issue by edurenye** [wadmiraal/docker-drupal#33]

**Comment by edurenye** [comment]
Maybe we could migrate to Alpine Linux as is built for docker, runs faster and has PHP 7.2

**Comment by wadmiraal** [comment]
Good idea. PR welcome. However, there's a lot to install for Alpine (PHPMyAdmin, Blackfire, SSH...). Not all of that is easy :-). The image must remain 100% BC.
